docs/content/latest/develop/graphql/hasura.md

```diff
@@ -37,26 +39,39 @@ For a local Mac setup, the configuration should be:
 
 ```sh
 docker run -d -p 8080:8080 \
-       -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@host.docker.internal:5433/postgres \
+       -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@host.docker.internal:5433/yugabyte \
        -e HASURA_GRAPHQL_ENABLE_CONSOLE=true \
-       hasura/graphql-engine:v1.0.0
+       hasura/graphql-engine:v1.1.0
 ```
 
 {{< note title="Note" >}}
 
-Make sure that the release version specified for `hasura/graphql-engine` matches the version you are using. The releases can be found at [Hasura graphql-engine releases](https://github.com/hasura/graphql-engine/releases).
+For the `hasura/graphql-engine` value, make sure that the version matches the Hasura GraphQL Engine version that you are using.
 
 {{< /note >}}
 
-To start Hasura, run the following script:
+After downloading or editing the `docker-run.sh` file, you need to make the file executable. Run the following `chmod` command:
```

Comment:
This feels out of place - it's not clear where the docker-run.sh file is coming from. Will be good to add more context like:
Make the `docker-run.sh` file downloaded from "Hasura Quick Start with Docker guide" executable by running the following `chmod` command.

Reply:
Good point — I have a major rewrite planned in the next few weeks, so I didn't get too picky with this update. I plan to do the tutorial from end to end, rather than send the user elsewhere as this does (most of this was written before me).

Reply:
Fixed and made the context clearer.